### ISSUE TITLE
field: separator:true  should be mentioned

### ISSUE THREAD

**Issue by Boorj** [bolt/docs#935]
There's an option `separator: true` which creates a Divider. It's being set up in field in contenttypes.yml. It's not documented, but it exists. Here's the code from `_field.twig`:

```twig

{# Postfix #}
{% if field.postfix is defined and field.postfix is not empty %}
    <div class="postfix">
        {{ field.postfix|markdown }}
    </div>
{% endif %}

{# Divider #}
{% if field.separator is defined and field.separator == true %}
    <hr>
{% endif %}
```
In docs there's even an advice to add <hr> manually.
> As you can see, using `postfix: "<hr>"` gives a simple and effective way of adding a divider in the edit screen.

So i think separator deserves to be documented. That could be fixed in docs, in _common_ and _nearby 'Postfix and prefix'_ section. And in cheatsheet too.

#### Places to edit:
**In docs:**
https://github.com/bolt/docs/blob/3.0/docs/fields/common.md#common-options-for-all-fields
and
https://github.com/bolt/docs/blob/3.0/docs/fields/common.md#prefix-and-postfix

**In cheatsheet:**
https://github.com/bolt/docs/blob/3.0/cheatsheet.yml#L135

**Comment by SvanteRichter** [comment]
I'm guessing a PR for this would be merged, and you already know the places to edit ;)

**Comment by Boorj** [comment]
@SahAssar i agree, but that was the point: that's such a responsibility to edit the documentation. So i made everything, that i could, to help some big boys to do that.

**Comment by bobdenotter** [comment]
Fixed in #940

**Comment by Boorj** [comment]
@bobdenotter now this PR seems to be overwritten
